integration-commons/src/test/java/org/symphonyoss/integration/exception/MissingConfigurationExceptionTest.java

```diff
@@ -0,0 +1,38 @@
+package org.symphonyoss.integration.exception;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link MissingConfigurationException}.
+ * Created by crepache on 23/06/17.
+ */
+public class MissingConfigurationExceptionTest {
+
+  @Test
+  public void testMissingConfigurationException() {
+    MissingConfigurationException exception = new MissingConfigurationException("component", "key");
+    String message = exception.getMessage();
+    String expected = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.